src/login/login.js

```diff
@@ -24,25 +39,37 @@ export default class Login extends Component {
                     style={styles.input}
                     value={this.state.username}
                     onChangeText={(text) => this.setState({username: text})}
-                    placeholder={'Enter User Nickname'}
+                    placeholder={'Enter Username'}
                     maxLength={12}
                     multiline={false}
                     />
+                  <TextInput
+                      style={styles.input}
+                      value={this.state.password}
+                      onChangeText={(text) => this.setState({password: text})}
+                      placeholder={'Enter Password'}
+                      maxLength={12}
```

Comment:
Right now we'll leave it open for navigating around, but I agree we may need to add a restriction here.